oshdb-tool/etl/README.md

```diff
@@ -22,7 +22,7 @@ mvn exec:java -Dexec.mainClass="org.heigit.ohsome.oshdb.tool.importer.extract.Ex
 ```
 
 This creates the files `extract_keys`, `extract_keyvalues` and `extract_roles` 
-containing the keys and the values of tags and the roles of relations.
+containing the keys, and the values of tags and the roles of relations.
```

Comment:
does this help making this more readable? maybe reformulate:

> This creates the files `extract_keys`, `extract_keyvalues` and `extract_roles` containing the keys of tags, the values of tags, and the roles of relations, respectively.

Reply:
rewritten